apps/datahub/src/app/record/record-linked-records/record-linked-records.component.ts

```diff
@@ -0,0 +1,42 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core'
+import { MdViewFacade } from '@geonetwork-ui/feature/record'
+import { map } from 'rxjs'
+import { RecordInternalLinksComponent } from '../record-internal-links/record-internal-links.component'
+import { CommonModule } from '@angular/common'
+import { TranslateModule } from '@ngx-translate/core'
+
+@Component({
+  selector: 'datahub-record-linked-records',
+  standalone: true,
+  imports: [CommonModule, TranslateModule, RecordInternalLinksComponent],
+  templateUrl: './record-linked-records.component.html',
+  styleUrl: './record-linked-records.component.css',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class RecordLinkedRecordsComponent {
+  sourceDatasets$ = this.metadataViewFacade.sources$
+  linkedDatasets$ = this.metadataViewFacade.sourceOf$.pipe(
+    map((records) => records?.filter((record) => record?.kind === 'dataset'))
+  )
+  linkedReuses$ = this.metadataViewFacade.sourceOf$.pipe(
+    map((records) => records?.filter((record) => record?.kind === 'reuse'))
+  )
```

Comment:
How about :

```suggestion
  linkedReuses$ = this.metadataViewFacade.sourceOf$.pipe(
  map((records) => records?.filter((record) => record?.kind === 'reuse') ?? [])
);
```

Instead of checking again in the getter?

Reply:
Thanks for the idea! Do you mean to check for the length in the template then?

Reply:
I would find that a bit more readable than having two checks in the logic but it's a preference :)

Reply:
Ok I kept it as is, as I think it'd still need the length check in the template if not